### ISSUE TITLE
Subject missing in jwt access tokens

### ISSUE THREAD

**Issue by davidgtonge** [panva/node-oidc-provider#303]
Hi,

Thanks for the new feature to support jwt access tokens.
Currently the tokens are issued without any `sub` claim. The claim can still be introspected via the introspection endpoint, but I would have thought that it should be included in the jwt to allow for key based verification rather than introspection, however maybe there is a reason that you don't expose the subject?

If it is an oversight, then it can be corrected with a single line:

```
jti, sub, iss, iat, exp, scope, aud, clientId: azp,
```

to

```
jti, accountId: sub, iss, iat, exp, scope, aud, clientId: azp,
```

in models/formats/jwt.js

**Comment by panva** [comment]
definitely an oversight, can you submit a fix PR with tests?

**Comment by davidgtonge** [comment]
ok, sure

**Comment by davidgtonge** [comment]
not sure where the best place to write tests for this is?
maybe in  `base_token.test.js`? why do you think?

**Comment by panva** [comment]
you can leave the tests to me, i'll create a test file for every format later.

**Comment by panva** [comment]
Ahh, initially opaque was remapping claims to be stored from accountId to sub, from clientId to azp. But I removed that requirement.

**Comment by panva** [comment]
Fixed in v4.1.1

**Comment by davidgtonge** [comment]
Thanks
